### DISCUSSION TITLE
New to Open Source

### DISCUSSION THREAD

**Discussion by Sabari2810** [github/india#48] [Open Source] [answered]
Hey, I am a developer, I've been developing and designing websites for almost two years now,  and I want to do something new, something like being part of an open-source project. I would really appreciate it, If  someone could help me with that. I just want to know how to actually be a part of an open-source project like Gatsby

**Comment by karthiksedoc** [comment, marked as answer]
The topic **An easier way to contribute to open source** in [GitHub Open Source](https://github.com/open-source) should get you started

**Reply by Sabari2810** [reply]
Thanks, Karthik for answering my question. I really appreciate it

**Reply by iamparasbansal** [reply]
Just to some Up everything, in a nutshell, spend time on exploring repositories of your worth as much as you can.
